fidelius-ui/src/app/add/add.component.spec.ts

```diff
@@ -41,6 +41,12 @@ describe('AddComponent', () => {
     createCredential(credential: Credential): any {
       return Observable.of({secret: false});
     }
+    
+    getSourceTypes(): any {
+      return Observable.of(
+        ["-", "Aurora", "RDS", "Service Account"]
```

Comment:
Not sure why we have this hardcoded.

Reply:
That's for the testing suite to work, that call needs to be mocked so I gave it a return value. I guess it could also just return an empty list